spec/controllers/user_journey_controller_spec.rb

```diff
@@ -175,6 +175,19 @@
       expect(subject).to render_template(:check_your_certificate)
     end
 
+    it 'renders index page when more than 2 signing certificates are trying to be submitted' do
+      signing_cert_primary = create(:msa_signing_certificate, component: msa_component)
+      signing_cert_secondary = create(:msa_signing_certificate, component: msa_component)
+      certmgr_stub_auth(team)
+      post :submit,
+          params: params.merge({
+            'upload-certificate': 'string',
+            certificate: { value: signing_cert_secondary.value, component: msa_component }
+          })
+      expect(response).to have_http_status(:success)
+      expect(subject).to render_template(root_path)
```

Comment:
can we also have an expectation that the certs doesn't get uploaded (i.e. the number of enabled certs is the same, and it did not make any change)